core/tests/protractor_utils/ContributorDashboardPage.js

```diff
@@ -50,14 +50,12 @@ var ContributorDashboardPage = function() {
     by.css('.protractor-test-suggestion-review-message'));
   var questionReviewModalHeader = element(by.css(
     '.protractor-test-question-suggestion-review-modal-header'));
-  var usernameContainer = element(by.css('.protractor-test-username'));
 
   var reviewRightsDiv = element(by.css('.protractor-test-review-rights'));
 
   this.get = async function() {
     await browser.get('/contributor-dashboard');
-    await waitFor.visibilityOf(
-      usernameContainer, 'Username takes too much time to load');
```

Comment:
The contributor dashboard page doesn't have "Loading ..." message at the outset so we need this check!

Note: `await waitFor.pageToFullyLoad();` only waits for "Loading ..." message to disappear.

Reply:
Done!